src/semgrep_agent/main.py

```diff
@@ -80,11 +65,8 @@ def main(
     )
 
     # Get Metadata
-    Meta = detect_meta_environment()
-    meta_kwargs = {}
-    if baseline_ref:
-        meta_kwargs["cli_baseline_ref"] = baseline_ref
-    meta = Meta(config, **meta_kwargs)
+    meta = generate_meta_from_environment(baseline_ref)
```

Comment:
Do we no longer allow baseline ref from the command line?

Reply:
There's no functionality change here afaict